opencl/source/sharings/gl/linux/gl_context_guard_linux.cpp

```diff
@@ -10,29 +10,8 @@
 
 namespace NEO {
 GLContextGuard::GLContextGuard(GLSharingFunctions &sharingFcns) : sharingFunctions(&sharingFcns) {
```

Comment:
is this class still needed ?

Reply:
Yes, it's used by the "common code" shared between linux / windows. The alternative would be `#ifdef` in the common code but I wanted to avoid touching windows / common code as much as possible.